shard/config/helper.py

```diff
@@ -7,20 +7,22 @@
 
 class ConfigHelper:
     @staticmethod
-    def generate_database_configs(unshard: Dict=None, shard: Dict=None) -> Dict:
+    def generate_database_configs(unshard: Optional[Dict]=None, shard: Optional[Dict]=None) -> Dict:
         configuration = {}
 
         # Make Unshard Configuration
         if unshard:
             for key, config in unshard.items():
-                configuration.update(make_replication_configuration(key=key, replication_config=config))
+                configuration.update(make_replication_configuration(
+                    key=key, master=config['master'], slaves=config.get('slaves', []), conn_max_age=config.get('conn_max_age', 0)
+                ))
 
         # Make Shard Configuration
         if shard:
             for shard_group, config in shard.items():
                 configuration.update(make_shard_configuration(
-                    shard_group=shard_group, shard_options=config['shard_options'], shards=config['shards'],
-                    db_options=config.get('db_options')
+                    shard_group=shard_group, database_name=config['database_name'], logical_count=config['logical_count'],
+                    conn_max_age=config.get('conn_max_age', 0), shards=config['shards'], options=config.get('options', {})
```

Comment:
config.get('conn_max_age', 0) <- be a const

Reply:
@antiline I did it.